models/classes/class.ConsumerService.php

```diff
@@ -30,6 +30,12 @@ class taoLti_models_classes_ConsumerService
 {
     const CLASS_URI = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LTIConsumer';
 
+    const PROPERTY_INSTANCE_CONSUMER_SERVICE = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#ServiceLtiConsumer';
```

Comment:
these 3 are not properties neither

Reply:
I have adjust the name and moved to other extension since it's used here (extension-tao-testlti)